models/template_files/fortran/printout.f

```diff
@@ -9,6 +9,7 @@
       subroutine printout
       implicit none
 
+      include '../vector.inc'
       include 'coupl.inc'
```

Comment:
same

Reply:
same as above, vector.inc is needed by coupl.inc